source/configure.rst

```diff
@@ -85,7 +85,7 @@ You may also configure the quipucords server by setting the following environmen
   - DJANGO_LOGGING_LEVEL
     - Infrastructure logging level
 
-To set these values, pass the ``docker run`` command the ``-e "NAME=VALUE`` argument.  For example to set ``QPC_SERVER_TIMEOUT`` do::
+To set these values for release 1.0.0, pass the ``docker run`` command the ``-e "NAME=VALUE`` argument.  For example to set ``QPC_SERVER_TIMEOUT`` do::
```

Comment:
To set these values, pass the  the ``-e "NAME=VALUE`` argument for the ``docker run`` command. For example, the following command shows how to set ``QPC_SERVER_TIMEOUT`` value, where 1.0.0 is the installed version of Quipucords::

NOTE the comma after "For example" phrase
"do?" no

Reply:
👍